libs/locales/bn-BD/apps-journeys-admin.json

```
@@ -181,7 +181,7 @@
   "Message Circle": "Message Circle",
   "Message Notify Circle": "Message Notify Circle",
   "Message Notify Square": "Message Notify Square",
-  "Message Sqaure": "Message Sqaure",
+  "Message Square": "Message Square",
```

Biome (2.1.2)

[error] The key Message Square was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)